documents/documentContentService.go

```diff
@@ -0,0 +1,53 @@
+package documents
```

Comment:
This file was renamed from `documentContentManager` -> `documentContentService`, as the underlying file created the Service.